### PR TITLE
feat(apu): improve bleed air EGT model

### PR DESCRIPTION
**Pull request by davidwalschots**
## Summary of Changes
In the real aircraft, using APU BLEED causes EGT to rise by more degrees than previously expected.
This PR improves the model with an EGT increase of 85~95 degrees celsius.
The EGT rise and fall also has a phase in which the rise/fall happens more rapidly than previously modelled.

Next to the above, I also adapted the existing startup EGT model to work correctly with lower OAT.

## Additional context

Discord username (if different from GitHub): SjotgunSjonnie#9623

## Testing instructions
Play with the APU BLEED and APU GEN! 😀

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: src/systems/systems/src/apu/APS3200.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ EGT effect (less air density causing higher EGT during start as it provides less
...
-1, 0;
-2, 0;
-3, 0;
-4, 0;
-5, 0;
-6, 0;
-7, 0;
-8, 0;
-9, 0;
+1, -85;
```
Comment (beheh) [inline src/systems/systems/src/apu/APS3200.md:11]: Negative 85 EGT? Is this some kind of corrected value?

Comment (davidwalschots) [reply src/systems/systems/src/apu/APS3200.md:11]: This data, when entered on http://polynomialregression.drque.net/online.php provides a function that will also work well when you for example start the APU at 40.000 feet with an OAT of -50. Before, the APU EGT would just skip to 0 immediately, which wasn't right.

We don't know (yet) what effect OAT and altitude have on the APU's EGT. Once we know their effect, we can make this formula more relative to those outside factors. Something for the future! 😀

Comment (beheh) [reply src/systems/systems/src/apu/APS3200.md:11]: I see! Sounds good.

**Review by beheh — APPROVED** [review]

**Comment by marcman86** [comment]
Category: QA Tester
Name: marcman86#4907
Date of testing: 1.3.2021
Version of the sim: 1.13.16.0
PR Tested: #3767
Tier of testing: 1
Changes to observe: APU EGT with bleed
Testing technique: Tryout

Steps made to test the Pull Request:

1.) Spawned C&D at EDDP
2.) Turned on bat. and ext. power
3.) Started APU

EGT (steady states): No bleed 345°C, with bleed 440°C

Issues: -

Overall Rating: Good

Conclusions of the testing: Difference of 95°. EGT is independent from TAT? 

Hope this helps :-)

**Comment by A320flyer** [comment]
Category: QA Tester/Trial
Name: A320Flyer #4650
Date of testing: Current date 02/03/2021
Version of the sim: 1.13.16.0
PR Tested: #3767
Tier of testing: 1
Changes to observe:
(to check changes with the APU and with bleed on and off)

Testing technique: Run tests of apu
Steps you've made to test the Pull Request:
1. Loaded up at KDEN in cold and dark state parked up at the gate
2. powered up the aircraft by turning the EXT power to own and the 2 batterys to on and waited for systems to boot up.
3. Started up the APU using the APU Master Switch and Start button then monitored the ECAM page for the APU and waited until it was steady
4. Pressed the APU Bleed button to on and watched for the change on the ECAM page until steady.


Flight notes (n/a):

Media: Insert screenshots, recordings, or anything relevant to the review here.

Issues (If Any):None
Steps to achieve the issue

Overall Rating: Good
Conclusions of the testing: Worked as should without bleed on i had a steady temp of 345C when i turned the bleed on stead temp of 430C which is a difference of 85C

**Pull request merged**